test/browser/column/renderEditCell.test.tsx

```diff
@@ -41,8 +41,9 @@ describe('Editor', () => {
   it('should open editor when user types', async () => {
     page.render(<EditorTest />);
     await userEvent.click(getCellsAtRowIndex(0)[0]);
-    await userEvent.keyboard('123{enter}');
-    expect(getCellsAtRowIndex(0)[0]).toHaveTextContent(/^1123$/);
+    // TODO: await userEvent.keyboard('123{enter}'); fails in FF
+    await userEvent.keyboard('{enter}123{enter}');
+    expect(getCellsAtRowIndex(0)[0]).toHaveTextContent('1123');
```

Comment:
We should revert all those back to strict checks

```suggestion
    expect(getCellsAtRowIndex(0)[0]).toHaveTextContent(/^1123$/);
```